gfreshii-webapp/src/main/java/com/edbootcamp/controller/AddUserController.java

```diff
@@ -0,0 +1,56 @@
+package com.edbootcamp.controller;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.simple.JSONObject;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+
+@Controller
+public class AddUserController {
+
+	@SuppressWarnings("unchecked")
+	@RequestMapping(value = "/createUser", method = RequestMethod.POST)
+	public ModelAndView addUser(@RequestParam("firstName") String firstName,
+								@RequestParam("lastName") String lastName,
+								@RequestParam("email") String email,
+								@RequestParam("userName") String userName,
+								HttpServletRequest request, HttpServletResponse response)
+	{
+		ModelAndView mv = new ModelAndView();
+		mv.setViewName("redirectPage");
+		mv.addObject("firstName", firstName);
+		mv.addObject("lastName", lastName);
+		mv.addObject("email", email);
+		mv.addObject("userName", userName);
+		
+		JSONObject jsonObj = new JSONObject();
+		
+		jsonObj.put("firstName", firstName);
+		jsonObj.put("lastName", lastName);
+		jsonObj.put("email", email);
+		jsonObj.put("userName", userName);
```

Comment:
Just bind all this to a User object, also Don't do this.  Remove this JSON mess AND clear out the dependency in your pom.xml.

Reply:
this was used to produce task 1 requirements, something quick. I eventually do plan on making this a full on pojo and add a user table with login functionality